GitExtUtils/ArgumentString.cs

```diff
@@ -1,24 +1,23 @@
-﻿using System;
-using System.Diagnostics;
-using GitCommands;
+﻿using System.Diagnostics;
 using JetBrains.Annotations;
 
-namespace GitUIPluginInterfaces
+namespace GitCommands
 {
     [DebuggerDisplay("{" + nameof(Arguments) + "}")]
     public readonly struct ArgumentString
     {
         [CanBeNull] public string Arguments { get; }
+        public int? Length { get => Arguments?.Length; }
```

Comment:
Suggest `public int Length => Arguments?.Length ?? 0;`, as from API stand point, a nullable length is quite unexpected.
`get => ` is unnecessary, it gets added under the cover.